mava/adders/reverb/base.py

```diff
@@ -74,6 +89,29 @@ def spec_like_to_tensor_spec(
     return tf.TensorSpec.from_spec(spec, name="/".join(str(p) for p in paths))
 
 
+def get_trajectory_net_agents(
+    trajectory: Union[Trajectory, mava_types.Transition], entry_net_keys: Dict[str, str]
+) -> Tuple[List, Dict[str, List]]:
+    """Returns a dictionary that maps network_keys to a list of agents using each
+    network key.
+
+    Args:
+        trajectory (Union[Trajectory, mava_types.Transition]): Trajectory recorded by
+        the adders.
+        entry_net_keys (Dict[str, str]): The network_keys used by each entry.
+    Returns:
+        agents (List): A sorted list of al the agent_keys.
+        trajectory_net_agents (Dict[str, List]): A dictionary that maps network_keys to
+        a list of agents using each network key.
+    """
+    agents = sort_str_num(trajectory.actions.keys())
```

Comment:
Thanks. I did check. The sort is necessary here because of the actions.keys. This could be in any order but the agents returned by this function need to be in the correct order.